.travis.yml

```diff
@@ -1,15 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - 6
+  - 8
 install:
   - travis_retry gem install s3_website -v 3.4.0
   - travis_retry pip install awscli --upgrade --user
-  - npm i -g yarn && yarn
+  - travis_retry npm ci
 before_script: webpack && npm test
 script: ./s3_deploy.sh
```

Comment:
Following our recent conventions, this should really be:
```
script: webpack && npm test
after_succes: ./s3_deploy.sh
```
and while you're at it you should probably add `dist: trusty`.